great-idea/css/index.css

```diff
@@ -49,16 +49,78 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+    box-sizing:border-box;
 }
 
 html, body {
     height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+	font-family: 'Titillium Web', sans-serif;
+	margin: auto;
+	width: 95%;
+	max-width: 1000px;
 }
 
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+.container{
+	width: 90;
+	margin: auto
+}
+
+.container header{
+	display: flex;
+	justify-content: space-between;
+	margin-top: 1px
+}
+
+ header nav {
+	display: flex;
+	justify-content: space-evenly;
+	width: 70%;
+}
+
+ header logo {
+	display: flex;
+}
+
+.cta{
+	display: flex;
+	justify-content: space-around;
+	margin-top: 10px;
+}
+
+.cta .cta-text{
+	display: flex;
+	flex-flow: column nowrap;
+	align-items: center;
+	line-height: 4rem;
+	font-size: 56px;
+}
+ .main-content .top-content{
```

Comment:
When we have repeats of the same statement you'll save some sanity by comma separating your rules.